index.bs

```diff
@@ -4177,10 +4285,17 @@ To <dfn>trigger aggregatable attribution</dfn> given an [=attribution trigger=]
     with |report| and |sourceToAttribute| is false:
     1. Return the [=triggering result=] ("<code>[=triggering status/dropped=]</code>",
         ("<code>[=trigger debug data type/trigger-aggregate-insufficient-budget=]</code>", null)).
+1. If |matchedBucket| is not null and the result of running [=check if an attribution source can create aggregatable contributions for matched bucket=]
```

Comment:
Shall we just pass `|matchedBucket|` to avoid invoking `find matching aggregatable bucket` twice?

Reply:
good point. Done.